baseapp/baseapp.go

```diff
@@ -41,13 +44,18 @@ const (
 	MainStoreKey = "main"
 )
 
+// StoreLoader defines a customizable function to control how we load the CommitMultiStore
+// from disk
+type StoreLoader func(ms sdk.CommitMultiStore) error
```

Comment:
maybe we add to the comment here mentioning it's utility for state migration?

Reply:
Improved godoc